packager/src/main/java/io/cdap/hub/publisher/GCSPublisher.java

```diff
@@ -186,11 +198,11 @@ private void putFile(String keyPrefix, File file) throws IOException {
         contentType = fileTypeMap.getContentType(file);
     }
 
-    String key = keyPrefix + file.getName();
+    Path key = resolveKeyForFile(keyPrefix, file);
```

Comment:
nit: I don't think we need a utility method just for this

Reply:
this line was duplicated in some other method too, that's why I refactored it in a method